docs/sources/flow/reference/components/otelcol.receiver.vcenter.md

```diff
@@ -0,0 +1,143 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/components/otelcol.receiver.vcenter/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/otelcol.receiver.vcenter/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/otelcol.receiver.vcenter/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.receiver.vcenter/
+title: otelcol.receiver.vcenter
+description: Learn about otelcol.receiver.vcenter
+labels:
+  stage: experimental
+---
+
+# otelcol.receiver.vcenter
+
+{{< docs/shared lookup="flow/stability/experimental.md" source="agent" version="<AGENT VERSION>" >}}
+
+`otelcol.receiver.vcenter` accepts metrics from a 
+vCenter or ESXi host running VMware vSphere APIs and
+forwards it to other `otelcol.*` components.
+
+> **NOTE**: `otelcol.receiver.vcenter` is a wrapper over the upstream
+> OpenTelemetry Collector `vcenter` receiver from the `otelcol-contrib`
+> distribution. Bug reports or feature requests will be redirected to the
+> upstream repository, if necessary.
+
+Multiple `otelcol.receiver.vcenter` components can be specified by giving them
+different labels.
+
+The full list of metrics that can be collected can be found [here][vcenter metrics].
```

Comment:
We avoid using the word "here" in links:
https://developers.google.com/style/link-text#write-link-text